test/distributions/test_distributions.py

```diff
@@ -4555,7 +4555,10 @@ def test_invalid(self):
         for Dist, params in BAD_EXAMPLES:
             for i, param in enumerate(params):
                 try:
-                    with self.assertRaises(ValueError):
+                    with self.assertRaisesRegex(
```

Comment:
Just seeing the test errors now -- the error message for arguments checking is different currently and only says that the parameter has incorrect value without specifying the support. So this regex match fails.

Reply:
oops, looks like I updated the wrong test; I'll fix shortly...